internal/database/sqlcommon/transaction_sql.go

```diff
@@ -49,23 +51,27 @@ func (s *SQLCommon) UpsertTransaction(ctx context.Context, transaction *fftypes.
 
 	// Do a select within the transaction to determine if the UUID already exists
 	transactionRows, _, err := s.queryTx(ctx, tx,
-		sq.Select("seq").
+		sq.Select("blockchain_ids").
 			From("transactions").
 			Where(sq.Eq{"id": transaction.ID}),
 	)
 	if err != nil {
 		return err
 	}
 	existing := transactionRows.Next()
-	transactionRows.Close()
 
 	if existing {
+		var existingBlockchainIDs fftypes.FFStringArray
+		_ = transactionRows.Scan(&existingBlockchainIDs)
```

Comment:
Should we check for error?

Reply:
... I took the approach that a problem here would be detected below, but it might have been lazy

Reply:
Yea, actually seems ok